pkg/cmd/server/api/validation/master.go

```diff
@@ -287,6 +287,18 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	if len(config.LoggingPublicURL) > 0 {
+		if _, loggingURLErrs := ValidateURL(config.LoggingPublicURL, "loggingPublicURL"); len(loggingURLErrs) > 0 {
```

Comment:
I made the logging route passthrough and enable the user to provide their own key/cert if desired so it can be as legitimate as necessary (I would hope at least some customers are signing certs from their corporate CA).  Of course the user can edit the route after the fact anyway.